lib/phil/products/importer.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Phil.Products.Importer do
+  @moduledoc """
+  Imports Products into Phil.
+  """
+
+  alias NimbleCSV.RFC4180, as: CSV
+
+  alias Phil.Products.Product
+  alias Phil.Repo
+
+  @doc """
+  Given the path of a file, this function attempts to import Products.
+
+  It returns a map of successes and failures like %{error: [], ok: []}.
+  Error entries are changesets and success entries are the inserted struct.
+  """
+  # sobelow_skip ["Traversal.FileModule"]
+  def import(path) do
+    path
+    |> File.read!()
+    |> CSV.parse_string(skip_headers: false)
+    |> Enum.map(&row_to_changeset/1)
+    |> Enum.map(fn changeset ->
+      if changeset.valid? do
+        Repo.insert(changeset)
```

Comment:
should it be an upsert to reduce the number of possible errors?

Reply:
Same thing, I don't think we should use import as an edit.